tests/api2/test_475_syslog.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from auto_config import dev_test, ip, password, user
+from functions import SSH_TEST
+from pytest_dependency import depends
+from time import sleep
+
+pytestmark = pytest.mark.skipif(dev_test, reason='Skipping for test development testing')
+
+
+def do_syslog(ident, message, facility='syslog.LOG_USER', priority='syslog.LOG_INFO'):
+    cmd = 'python3 -c "import syslog;'
+    cmd += f'syslog.openlog(ident=\\\"{ident}\\\", facility={facility});'
+    cmd += f'syslog.syslog({priority},\\\"{message}\\\");syslog.closelog()"'
+    results = SSH_TEST(cmd, user, password, ip)
+    assert results['result'] is True, results['output']
+
+
+def check_syslog(log_path, message):
+    cmd = f'grep -R "{message}" {log_path}'
+    results = SSH_TEST(cmd, user, password, ip)
+    assert results['result'] is True, results['output']
+
+
+def validate_syslog_filter(data):
```

Comment:
Why define a function only used once?

Reply:
That's a fair criticism. I wrote the functions before writing tests.